elements/rh-pagination/rh-pagination.css

```diff
@@ -0,0 +1,89 @@
+:host {
+  display: block;
+  min-height: 4em;
+}
+
+[hidden] {
+  display: none !important;
+}
+
+#container {
+  display: flex;
+  gap: calc(var(--rh-space-xs, 4px) / 2);
+  flex-flow: row wrap;
+}
+
+nav {
+  display: contents;
+}
+
+svg {
+  fill: currentcolor;
+}
+
+.stepper {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 50px;
+  width: 50px;
+  background: var(--rh-color-surface-light, #f0f0f0);
+  outline: none;
+  color: var(--rh-pagination-stepper-color, var(--rh-color-black-600, #6a6e73));
+  box-sizing: border-box;
+}
+
+.stepper:focus {
+  border: 2px solid var(--rh-color-border-interactive-on-light, #0066cc);
+}
+
+.stepper svg {
+  height: 14px;
+}
+
+:is(#next, #last) svg {
+  rotate: 180deg;
+}
+
+.stepper[inert] {
+  pointer-events: none;
+  color: var(--rh-pagination-background-focused, var(--rh-color-black-300, #d2d2d2));
+}
+
+/* MOBILE styles */
+
+.mobile .stepper:hover {
+  background: var(--rh-color-black-300, #d2d2d2);
+}
+
+#numeric {
+  margin-block-start: var(--rh-space-md, 8px);
+  width: 100%;
+  display: flex;
+  align-items: center;
+  gap: 0.5em;
+}
+
+input {
+  height: var(--rh-length-lg, 16px);
+  width: var(--rh-length-xl, 24px);
+  background: var(--rh-color-surface-lightest, #ffffff);
+  border: 1px solid var(--rh-color-black-300, #d2d2d2);
+  border-block-end: 1px solid var(--rh-color-black-600, #6a6e73);
+  box-sizing: border-box;
+  padding: 2px;
```

Comment:
Width should be 64px, and height should be 32px.
Font size should be 16px.

The linked number after "of" should have #06c for the text color and no underline.

Reply:
ok done, but for the colour, I improvised the hover and visited states, PTAL